src/pages/api/refresh.js

```diff
@@ -0,0 +1,24 @@
+import glob from 'glob'
+
+function getPair(path) {
+  let arr = path.split('/')
+  let comp = arr.pop()
+  comp = comp.split('.')[0]
+  let dir = arr.join('/')
+  return { comp, dir }
+}
+
+async function getJsons() {
+  let jsons = {}
+  const files = glob.sync('../remote/tu2k22-chakra/**/*.oc.json', {})
```

Comment:
tu2k22 seems to be hard-coded

Reply:
will use the solution that will only work for devspaces for now, I'll return to this later